dev/EnvironmentManager/API/Microsoft.Windows.System.EnvironmentManager.Insights.h

```diff
@@ -0,0 +1,45 @@
+﻿#pragma once
+#include "Microsoft.Windows.System.EnvironmentManager.h"
+
+class EnvironmentManagerInsights : public wil::TraceLoggingProvider
+
+{
+    IMPLEMENT_TRACELOGGING_CLASS(
+        EnvironmentManagerInsights,
+        "Microsoft.Windows.System.EnvironmentManager.Insights",
+        // {fbce41fa-f7bc-538e-11f2-abe1b9d2ab23}
+        (0xfbce41fa, 0xf7bc, 0x538e, 0x11, 0xf2, 0xab, 0xe1, 0xb9, 0xd2, 0xab, 0x23));
+
+    DEFINE_COMPLIANT_MEASURES_EVENT_STRING(StringTelemetryEvent, // Event name 
+        MICROSOFT_KEYWORD_MEASURES, // Check wil/traceloggingconfig.h for a full list 
+        stringvalue/* value to be logged*/);
+
+    DEFINE_COMPLIANT_TELEMETRY_EVENT_PARAM2(ScopeTelemetryEvent, //Event name
```

Comment:
>ScopeTelemetryEvent [](http://example.com/codeflow?start=44&length=19)

Where is this event being logged? Isn't this event identical to the LogWithScopeAndMessage function with the difference that LogWithScopeAndMessage converts the scope to string while this logs the scope as an enum.

Reply:
You commented before I pushed. 

This was an attempt to log the scope and string before I moved onto the custom event.  It is gone now and replaced with a customer even "LogMessageAndScope"